Core/Object Arts/Dolphin/Base/MessageSendAbstract.cls

```diff
@@ -99,6 +99,28 @@ receiver: anObject
 
 	^self subclassResponsibility!
 
+storeOn: aStream
+
+	aStream
+		nextPut: $(;
+		nextPutAll: self class name;
+		nextPutAll: ' receiver: '.
+	self receiver storeOn: aStream.
+	aStream 
+		nextPutAll: ' selector: ';
+		print: selector.
+	0 < self argumentCount ifTrue: [
```

Comment:
General comment: We don't use this c-style inverted comparison. New code needs to be consistent with the existing conventions